src/app/desktop/views/Home.js

```diff
@@ -28,18 +28,26 @@ const Home = () => {
   const dispatch = useDispatch();
   const account = useSelector(_getCurrentAccount);
   const news = useSelector(state => state.news);
+  const isNewUser = useSelector(state => state.app.isNewUser);
 
   const openAddInstanceModal = defaultPage => {
     dispatch(openModal('AddInstance', { defaultPage }));
   };
 
+  const openWelcomeModal = () => {
+    dispatch(openModal('Onboarding'));
+  };
+
```

Comment:
What's the point of this function? It's used in only 1 place and it does only one thing